tests/cbmc/.gitignore

```diff
@@ -5,3 +5,6 @@ gotos
 html
 logs
 property.xml
+
+arpa_cmake
+Makefile.arpa
```

Comment:
This is good, but I suggest that you also add these files to the [template gitignore in this directory](https://github.com/awslabs/aws-templates-for-cbmc-proofs/tree/master/template-for-repository) (not the top-level one). That gitignore gets copied into projects that use the template.